core/src/main/java/software/amazon/awssdk/core/util/UserAgentUtils.java

```diff
@@ -72,7 +72,7 @@ static String userAgent() {
 
         ua = ua
                 .replace("{platform}", "java")
-                .replace("{version}", VersionInfo.SDK_VERSION)
+                .replace("{version}", "MOCK")//VersionInfo.SDK_VERSION)
```

Comment:
FIXME: had a compile error here, remove in case we'd intend to merge this PR

Reply:
If you run a `mvn generate-sources -pl :core` it should generate the `VersionInfo` class for you and add it to the generated-sources directory.

Reply:
Thanks!